### PR TITLE
#350 Add the git hash to the log header

### LINKED ISSUE #350: [Compilation] Generate the git version file and add it to the log header to homogenize with others projects

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #350

### CODE REVIEW

File: cmake/getgitheadrevision.cmake [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+
+include(FindGit)
+find_package(Git)
+
+message(INFO "BUBU ${OUTPUT_DIR}")
```
Comment (marcochiaramello) [inline cmake/getgitheadrevision.cmake:12]: BUBU ? :)

File: sources/CMakeLists.txt [hunk 2]
```diff
@@ -42,6 +43,21 @@ target_link_libraries(DynaFlowLauncher
   LibXml2::LibXml2
 )
 
+# explicitly say that the executable depends on custom target
+add_dependencies(DynaFlowLauncher revisiontag)
+
+#obtain git revision
```
Comment (marcochiaramello) [inline sources/CMakeLists.txt:49]: space after  #

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: cmake/getgitheadrevision.cmake (hunk 1 above)
Comment (rosiereflo) [reply cmake/getgitheadrevision.cmake]: solved

File: sources/CMakeLists.txt (hunk 2 above)
Comment (rosiereflo) [reply sources/CMakeLists.txt]: solved

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**